make_profiler/report/index.html

```diff
@@ -0,0 +1,23 @@
+<html>
+
+<head>
+    <title>Kontur Make Profiler Report</title>
+    <link rel="icon" href="favicon.png" type="image/x-icon">
+    <link rel="stylesheet" href='report/style.css'>
+    <link href='https://fonts.googleapis.com/css?family=IBM Plex Sans Condensed' rel='stylesheet'>
+    <script src='report/script_u.js' type="text/javascript"> </script>
+
+    <script>
+        window.onload = function () {
+            var tag = document.createElement("script");
+            tag.src = "report/sorttable.js";
+            document.getElementsByTagName("head")[0].appendChild(tag);
+        }
+    </script>
+</head>
+
+<body>
+    <div id="status">Either not using a web server or can not reach status report details!</div>
```

Comment:
`Either not using a web server` will it not work if browsed locally?

Reply:
Kirill copies related files to report folder under Geocint folder and CORS policy doesn't allow reaching json.

Reply:
if the files are copied by Python anyway how about just embedding the json into the html? 

or using not a json, but .js with `data = {...};` inside and plugging it as good old script?

also internet suggests that fetch api works on local files (haven't checked).

Reply:
Used fetch api. Made changes to handle errors inside catch.